admin-dev/themes/default/template/controllers/login/header.tpl

```diff
@@ -49,6 +49,6 @@
 		{/if}
     <script type="text/javascript" src="../js/admin/login.js?v={$smarty.const._PS_VERSION_|escape:'html':'UTF-8'}"></script>
 	</head>
-	<body class="ps_back-office bootstrap">
+	<body class="lang-{$iso_user}{if $lang_is_rtl} lang-rtl{/if} ps_back-office bootstrap">
```

Comment:
Is this part really necessary?
```suggestion
	<body class="{if $lang_is_rtl} lang-rtl{/if} ps_back-office bootstrap">
```

Reply:
Not really, I just copied from new theme :)
`{if $lang_is_rtl} lang-rtl{/if}` is enough.

Reply:
Maybe we should remove it from new-theme too?